lib/main_la.dart

```diff
@@ -1,7 +1,13 @@
+import 'package:dart_wormhole_william/client/native_client.dart';
 import 'package:destiny/constants/app_constants.dart';
 
 import 'main.dart';
 
 void main() {
+  final Config leastAuthority = Config(
+    rendezvousUrl: "wss://mailbox.mw.leastauthority.com/v1",
```

Comment:
other places, we've talked about standardizing on "mailbox" for for the name of this

Reply:
That is not part of this PR. If we have a decision we can update the URL. Not before.

Reply:
Well, we do have a decision ...and burning "rendezvous" into a bunch more places seems like it _is_ appropriate to mention here, since it's refactoring exactly that..?

Reply:
At first I am not aware that we have a decision, please have a look at the thread in Slack: https://leastauthority.slack.com/archives/C04BZDSL4UX/p1670237726671269

Then please have a closer look at the code:
1. `Config` is declared here https://github.com/LeastAuthority/dart_wormhole_william/blob/main/lib/client/native_client.dart and
2. no, it is not user facing.

We need to rename the field when we have a decision. This will change only a field which is not visible to the user.

Reply:
The `Config` class containing the field `rendezvousUrl` was not added with this PR (Have a look at `lib/constants/app_constants.dart` for reference).
The class `Config` itself is defined in https://github.com/LeastAuthority/dart_wormhole_william which is included in a Git submodule.
This PR contains a refactoring to limit internal use to the extent that is still required to be functional.
Next step is to fix that `rendezvousUrl` name in the dependency. Then we can remove those references here.

Reply:
Have created a ticket to fix that upstream: https://github.com/LeastAuthority/dart_wormhole_william/issues/43